lib/config/presets/github/index.ts

```diff
@@ -15,9 +15,7 @@ export async function fetchJSONFile(
   const url = `${endpoint}repos/${repo}/contents/${fileName}`;
   const opts: HttpOptions = {
     headers: {
-      accept: global.appMode
-        ? 'application/vnd.github.machine-man-preview+json'
-        : 'application/vnd.github.v3+json',
+      accept: 'application/vnd.github.v3+json',
```

Comment:
I'm thinking that accept headers could be better achieved using hostRules instead.

Reply:
maybe remove this if already done in util/http